libs/ui-lib/lib/ocm/components/clusterWizard/disconnected/BasicStep.tsx

```diff
@@ -85,21 +85,26 @@ const BasicStep = () => {
         <WithErrorBoundary title="Failed to load Basic step">
           <Grid hasGutter>
             <GridItem>
-              <Split>
-                <SplitItem>
-                  <Content component="h2">Basic information</Content>
-                </SplitItem>
-                <SplitItem>
-                  <DeveloperPreview />
-                </SplitItem>
-              </Split>
+              <Content component="h2">Basic information</Content>
             </GridItem>
             <GridItem>
-              <InstallDisconnectedSwitch />
+              <div
+                style={{
+                  display: 'flex',
+                  alignItems: 'center',
+                  gap: 'var(--pf-t--global--spacer--sm)',
+                }}
+              >
+                <TechPreview />
+                <InstallDisconnectedSwitch />
+                <span>
+                  {t("ai:I'm installing on a disconnected/air-gapped/secured environment")}
+                </span>
+              </div>
             </GridItem>
             <GridItem>
               <Form id="wizard-cluster-basic-info__form">
-                <OcmOpenShiftVersion openshiftVersion="4.19" withPreviewText withMultiText>
+                <OcmOpenShiftVersion openshiftVersion="4.20" withPreviewText withMultiText>
```

Comment:
Instead of hardcoding, can the version be extracted from the release image or any other way instead of hardcoding here?